test/weighted_median.cpp

```diff
@@ -41,14 +41,21 @@ void test_stat()
     for (std::size_t i=0; i<100000; ++i)
```

Comment:
you can increase the number of iteration to 1,000,000. and the test would pass (the problem is with the 3% requirement).

Reply:
Okay, I'll give it a whirl.

Reply:
p_square_cumul_dist.cpp seems to suffer from the same problem. I have it passing with
```diff
diff --git a/test/p_square_cumul_dist.cpp b/test/p_square_cumul_dist.cpp
index 4bc014d..57d3f8c 100644
--- a/test/p_square_cumul_dist.cpp
+++ b/test/p_square_cumul_dist.cpp
@@ -51,7 +51,7 @@ void test_stat()
     boost::normal_distribution<> mean_sigma(0,1);
     boost::variate_generator<boost::lagged_fibonacci607&, boost::normal_distribution<> > normal(rng, mean_sigma);
 
-    for (std::size_t i=0; i<100000; ++i)
+    for (std::size_t i=0; i<1000000; ++i)
     {
         acc(normal());
     }
```

Reply:
Finally, success!